service-deployment/bootstrap/webportal/webportal-ds.yaml.template

```diff
@@ -33,6 +33,9 @@ spec:
       containers:
       - name: webportal
         image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}webportal
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
```

Comment:
Does this mean the docker command will use the `--privileged` parameter? I heard from @fanyangCS that this is not secure.